automation/postprocess/spectrogram.py

```diff
@@ -0,0 +1,15 @@
+import subprocess
+
+from .commons import input_output_parser
+
+def create_spectogram(input_path: str, output_path: str):
+    subprocess.check_call(["sox", input_path, "-n", "spectrogram", "-o", output_path])
+
+if __name__ == '__main__':
```

Comment:
I'm not able to run this script. All attempts to call it failed miserably:
```
$ python postprocess/spectrogram.py 
Traceback (most recent call last):
  File "postprocess/spectrogram.py", line 3, in <module>
    from .commons import input_output_parser
ImportError: attempted relative import with no known parent package
```

Reply:
Fixed.